extensions/indexes/lucene/src/test/java/org/exist/indexing/lucene/LuceneIndexTest.java

```diff
@@ -363,13 +363,13 @@ public void moreElaborateQueries() throws EXistException, CollectionConfiguratio
             };
 
             String[]  queryTemplates = {
-                    "//.[ft:query(title, '%s')]",
```

Comment:
`//.` is a valid XPath expression after all. This means there is a regression that needs fixing.